lib/permut.gi

```diff
@@ -2,7 +2,7 @@
 ##
 #W  permut.gi              Permutability GAP library              ABB&ECL&RER
 ##
-#H  @(#)$Id: permut.gi,v 0.04 2014/01/16 09:53:49 gap Exp $
+#H  @(#)$Id: permut.gi,v 1.04 2018/02/11 19:36:48 gap Exp $
```

Comment:
There is no need in this line any more - may be deleted (and in other files too). This is a relic from CVS times. CVS was able to put timestamps automatically in files. Now this info may be recovered from the repository revisions history.